app/models/user.rb

```diff
@@ -3,4 +3,5 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
+  has_many :likes
```

Comment:
Should likes be also removed if a user record is deleted? Use dependent destroy in that case.